src/prefect/server/database/configurations.py

```diff
@@ -300,6 +315,8 @@ async def engine(self) -> AsyncEngine:
                 # that a given connection pulled from the pool will be
                 # usable.
                 pool_use_lifo=True,
+                json_serializer=orjson_dumps,
+                json_deserializer=orjson_loads,
```

Comment:
no risk with this change?

Reply:
yea let's separate it out so we can revert it easily if we need to / can study it a little more just in case